server/app.ts

```diff
@@ -0,0 +1,82 @@
+
+import express,{NextFunction, Request,Response} from 'express';
+import swaggerJsdoc from 'swagger-jsdoc';
+import swaggerUi from 'swagger-ui-express';
+import dotenv from 'dotenv'
+dotenv.config({path:__dirname+'/../.env'});
+import sequelize, { items } from './database';
```

Comment:
I think this proving that there are no eslint rules, sometimes it is  {items},  sometimes { items }